sdk/servicebus/azure-messaging-servicebus-track1-perf/src/main/java/com/microsoft/azure/servicebus/perf/ReceiveMessagesTest.java

```diff
@@ -0,0 +1,79 @@
+package com.microsoft.azure.servicebus.perf;
+
+import com.azure.perf.test.core.TestDataCreationHelper;
+import com.microsoft.azure.servicebus.IMessage;
+import com.microsoft.azure.servicebus.Message;
+import com.microsoft.azure.servicebus.perf.core.ServiceBatchTest;
+import com.microsoft.azure.servicebus.perf.core.ServiceBusStressOptions;
+import reactor.core.publisher.Mono;
+
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.List;
+import java.util.UUID;
+import java.util.concurrent.CompletableFuture;
+
+public class ReceiveMessagesTest extends ServiceBatchTest<ServiceBusStressOptions> {
+    /**
+     * Creates an instance of Batch performance test.
+     *
+     * @param options the options configured for the test.
+     * @throws IllegalStateException if SSL context cannot be created.
+     */
+    public ReceiveMessagesTest(ServiceBusStressOptions options) {
+        super(options);
+    }
+
+    @Override
+    public int runBatch() {
+        Collection<IMessage> messages;
+        try {
+            messages = receiver.receiveBatch(options.getMessagesToReceive());
+            if (messages.size() <= 0) {
+                throw new RuntimeException("Error. Should have received some messages.");
+            }
+        } catch (Exception e) {
```

Comment:
Is this Exception catch needed ?
Can we capture a specific exception if being thrown by the API ?

Reply:
I just copied and pasted the track1 old test. After dig deep into the code, the exception catch can be removed.